test/helper.rb

```diff
@@ -23,4 +23,4 @@ def method_missing(method, *args)
 end
 
 require "fluent/plugin/out_map"
-require "fluent/plugin/filter_map" if defined?(Fluent::Filter)
+require "fluent/plugin/filter_map" if defined?(Fluent::Plugin::Filter)
```

Comment:
No. Keep out_map plugin for now.
`Fluent::Plugin::Filter` is defined always with Fluentd v0.14.